Contact_Us/contact.html

```diff
@@ -68,8 +68,11 @@
     <div class="contactDetails">
       <span>
         <h2><i class="fas fa-paper-plane"></i> Contact Us</h2>
-        <p>Lorem ipsum, dolor sit amet consectetur adipisicing elit. Eius soluta eveniet delectus nemo, reiciendis
-          repellendus.</p>
+        <p>you can contact our team using our
+          <a href="mailto:bteasean@gmail.com">official email
+            <a href="https://discord.gg/thA62UNzRy">or our discord server.</a> 
```

Comment:
Add the `anchor tag` only to the term `discord` not entirely to `or our discord server`

Reply:
yessir!